kustomize/components/postgres/postgres.deployment.yaml

```diff
@@ -6,8 +6,6 @@ metadata:
     quay-component: postgres
 spec:
   replicas: 1
-  strategy:
```

Comment:
Note: the Postgres pod will still be recreated, but changing the strategy to `type: RollingUpdate` and persisting the password across reconciles means the database will remain available.